eng/common/pipelines/templates/steps/docindex.yml

```diff
@@ -49,6 +47,7 @@ jobs:
           targetPath: $(Build.ArtifactStagingDirectory)/docfx_project/_site
           
       - pwsh: |
+          git commit -am "Save the language updates."
```

Comment:
Is the commit only for the mutated files or is it also for the generated files?

Reply:
Only for mutated files.